packages/shared/ReactTypeOfWork.js

```diff
@@ -26,11 +26,11 @@ export type TypeOfWork =
   | 15
   | 16;
 
-export const IndeterminateComponent = 0; // Before we know whether it is functional or class
-export const FunctionalComponent = 1;
-export const FunctionalComponentLazy = 2;
-export const ClassComponent = 3;
-export const ClassComponentLazy = 4;
+export const FunctionalComponent = 0;
+export const FunctionalComponentLazy = 1;
+export const ClassComponent = 2;
```

Comment:
This should have a big bold inline comment explaining the significance of 2

Reply:
I don't know why I added the word "bold" 🤡

Reply:
I think it'll be hard to miss any changes because they broke so much in internal tests (e.g. enzyme). We're also already very intentional about changing these constants.

If we merge this, it'll be just a temporary concession. We still have to "get our act together" later.

Reply:
I still think an inline comment might be helpful to avoid situations like the one that just happened (person A changing this, and person B discovering a few days later that it causes a lot of churn)

Reply:
I think the one that just happened seemed pretty intentional 😈 We should still fix this. If you feel strongly please feel free to send PR with a comment you'd find valuable. Maybe something like `// Dependencies: *`. But I'm also wary of making it seem like these are untouchable.